RDITemplateMaster.dld

```diff
@@ -8080,5 +8172,5 @@ Do
   SDI12SensorResponse(IridiumOutArray)'sensor will respond with entire array when Iridium Radio queries DCP
 Loop
 #EndIf
-
-  EndProg
+      NextScan
```

Comment:
Next scan need to occur before the conditionally compiled iridium slow sequence section. End program occurs after the slow sequence.

Reply:
Addressed. Setting this to "Start a Review" to see how it works.